components/sync/driver/brave_sync_service_impl.cc

```diff
@@ -62,8 +62,15 @@ bool BraveSyncServiceImpl::IsSetupInProgress() const {
 }
 
 void BraveSyncServiceImpl::StopAndClear() {
-  SyncServiceImpl::StopAndClear();
+  // Suspend/resume actually does the code flow shorter, avoiding double nested
```

Comment:
do we still need these after using `HandleReset`?

Reply:
I have double checked - no we don't need.
With the latest changes there is no extra nested call of `SyncServiceImpl::StopAndClear()`. Thanks for this notice.

Fixed at https://github.com/brave/brave-core/pull/16117/commits/ef372da092664e4e2ca96eac95aa62ae62a6e4a8